base/models/models.go

```diff
@@ -139,6 +139,14 @@ type AdvisoryType struct {
 	Preference int
 }
 
+var AdvisoryTypes = map[int]string{
+	0: "unknown",
+	1: "enhancement",
+	2: "bugfix",
+	3: "security",
+	4: "unspecified",
+}
```

Comment:
You should read this enumeration from database not hardcode it.

Reply:
I wanted to discuss this one with you. I've seen we read those values for each request on some endpoint. These values should not change during runtime so wouldn't it be better if we used singleton to read from instead? In similar way as config vars. What do you think @MichaelMraka ?

Reply:
And to be clear reading these from DB for each notification push is not a problem at all, so we can discuss about it later as well if you prefer :wink:

Reply:
Already changed for querying DB for now and lets talk about it later in improvement scope.